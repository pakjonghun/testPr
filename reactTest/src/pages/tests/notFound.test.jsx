import renderer from "react-test-renderer";
import NotFound from "../NotFound";
import { Route } from "react-router-dom";
import { withRoute } from "../../tests/utils";

describe("not found page", () => {
  it("정확하게 페이지가 랜더링 되야 한다", () => {
    const tree = renderer
      .create(withRoute(<Route path='/' element={<NotFound />} />))
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
