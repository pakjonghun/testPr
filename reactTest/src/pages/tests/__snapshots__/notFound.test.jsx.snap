// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`not found page 정확하게 페이지가 랜더링 되야 한다 1`] = `
<div>
  Not Found!
</div>
`;
